### PR TITLE
feat: don't fix eslint at ci

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines
##### Description of change

- only fix eslint at `npm run test`
- don't fix eslint at `npm run ci`
- related: https://github.com/eggjs/egg/pull/100

### CODE REVIEW

File: boilerplate/package.json [hunk 1]
```diff
@@ -26,10 +26,10 @@
   "scripts": {
     "start": "node index.js",
     "dev": "egg-bin dev",
-    "test": "npm run lint && npm run test-local",
+    "test": "npm run lint -- --fix && npm run test-local",
```
Comment (luckydrq) [inline boilerplate/package.json:29]: 这里需要搞两个`--`?

Comment (atian25) [reply boilerplate/package.json:29]: 嗯, 这样才能传递进去. 看 https://docs.npmjs.com/cli/run-script

Comment (luckydrq) [reply boilerplate/package.json:29]: 👍

**Comment by popomore** [comment]
+1

**Comment by luckydrq** [comment]
另外，我看有个`--cache`的选项，貌似可以提升点性能，可以的话我PR

**Comment by atian25** [comment]
cache 什么？

**Comment by luckydrq** [comment]
会记录文件的mtime等信息以及结果，如果下次lint时文件没改动，会跳过lint过程。没仔细研究原理，在我的一个项目里试了下还不错。

------------------------------------------------------------------发件人：TZ | 天猪notifications@github.com日　期：2016年09月12日 18:19:45收件人：eggjs/egg-boilerplate-simpleegg-boilerplate-simple@noreply.github.com抄　送：Deng Ruoqidrqzju@gmail.com; Commentcomment@noreply.github.com主　题：Re: [eggjs/egg-boilerplate-simple] feat: don't fix eslint at ci (#12)cache 什么？

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/eggjs/egg-boilerplate-simple","title":"eggjs/egg-boilerplate-simple","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/eggjs/egg-boilerplate-simple"}},"updates":{"snippets":[{"icon":"PERSON","message":"@atian25 in #12: cache 什么？"}],"action":{"name":"View Pull Request","url":"https://github.com/eggjs/egg-boilerplate-simple/pull/12#issuecomment-246306736"}}}

**Comment by atian25** [comment]
低频命令，本地无所谓，可以加上。 ci 还是不要开的好。

**Comment by atian25** [comment]
倒是可以给IDE的lint配置加上

**Comment by luckydrq** [comment]
ci加不加也无所谓，.eslintcache不提交的

**Pull request merged**